File: Python/Recursion-Programs/43-Nth-fibonacci-number.py
num = int(input("Enter N: "))

def Fibonacci(num):
    if num == 0:
        return 0
    elif num == 1:
        return 1
    else:
        return Fibonacci(num-1) + Fibonacci(num-2)

print("\nNth Fibonaci number:", Fibonacci(num))

File: Java/3-Recursion-Programs/43-Nth-fibonacci-number.java
import java.util.Scanner;

class Main {
    static int Fibonacci(int num) {
        if (num == 0){
            return 0;
        } else if (num == 1) {
            return 1;
        } else {
            return Fibonacci(num-1) + Fibonacci(num-2);
        }
    }

    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.println("Enter N:");
        int num = getInput.nextInt();

        System.out.println("\nNth Fibonacci number: " + Fibonacci(num));
        getInput.close();
    }
}